tests/models/blenderbot_small/test_modeling_blenderbot_small.py

```diff
@@ -242,10 +242,7 @@ class BlenderbotSmallModelTest(ModelTesterMixin, GenerationTesterMixin, Pipeline
     def is_pipeline_test_to_skip(
         self, pipeline_test_casse_name, config_class, model_architecture, tokenizer_name, processor_name
     ):
-        if pipeline_test_casse_name == "TextGenerationPipelineTests":
-            return True
-        # TODO @Rocketnight1 to fix
-        if pipeline_test_casse_name == "ConversationalPipelineTests":
+        if pipeline_test_casse_name in ("TextGenerationPipelineTests", "ConversationalPipelineTests"):
             return True
         return False
```

Comment:
```suggestion
        return pipeline_test_casse_name in ("TextGenerationPipelineTests", "ConversationalPipelineTests")
```

Reply:
@amyeroberts That was putting that way for 2 (minor) reasons:

  - it's likely the condition could get more branches: if further failing tests happens or new pipeline test classes are added
  - make it clearly that, by default, it should return `False`: so no test should be skipped

But good for me if you prefer to apply the suggested changes

Reply:
I don't feel very strongly - they were just suggestions. Happy to leave as-is :)

Reply:
I like the suggestion, so I'll take it! We can always edit later if we need to change the condition.